packages/inspection-capture-web/test/PhotoCapture/PhotoCaptureHUD/PhotoCaptureTutorial/PhotoCaptureHUDTutorial.test.tsx

```diff
@@ -0,0 +1,147 @@
+jest.mock(
+  '../../../../src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDElementsSight/SightGuideline',
+  () => ({
+    SightGuideline: jest.fn(() => <></>),
+  }),
+);
+
+import { render, screen } from '@testing-library/react';
+import { Button, DynamicSVG } from '@monkvision/common-ui-web';
+import {
+  PhotoCaptureHUDTutorial,
+  PhotoCaptureHUDTutorialProps,
+  SightGuideline,
+} from '../../../../src';
+import { expectPropsOnChildMock } from '@monkvision/test-utils';
+import { TutorialSteps } from '../../../../src/PhotoCapture/hooks';
+
+const BACKDROP_TEST_ID = 'backdrop';
+const TITLE_TEST_ID = 'title';
+const translationPrefix = 'photo.hud.tutorial';
+
+function createProps(): PhotoCaptureHUDTutorialProps {
+  return {
+    tutorialStep: TutorialSteps.WELCOME,
+    onNextTutorialStep: jest.fn(),
+    onCloseTutorial: jest.fn(),
+    allowSkipTutorial: false,
+    sightGuidelines: [],
+    sightId: 'test-sight-id',
+  };
+}
+
+describe('PhotoCaptureHUDTutorial component', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should not be shown if tutorialView is null', () => {
```

Comment:
Change `tutorialView` to `tutorialStep` in the test name (and also in all other test names in this file)

Reply:
@dlymonkai this is not fixed